pages/api/check-verification-status.ts

```diff
@@ -19,12 +24,12 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
 
     const data = await result.json();
     if (data.status === "1") {
-      return res.status(200).json({ result: data.result });
+      return NextResponse.json({ result: data.result }, { status: 200 });
```

Comment:
in this case we should add a cache flag here (we want the client to cache this response forever really, a contract can never be un-verified)

Reply:
Done